spec/macros/resource_macros.rb

```diff
@@ -110,7 +114,7 @@ def it_should_be_deletable(options = {})
           if options[:find]
             expect(obj.send(options[:find].first)).to eq(options[:find].last)
           else
-            expect(obj).to be_nil
+            expect(obj.nil? || !obj.active?).to eq(true)
```

Comment:
can you explain what's going on here?

Reply:
Can you (@fbvilela), test if the `random_string` fix works without this change?

Reply:
Casper, it didn't work. 

This change is because `OrganizationField` is marked for deletion and returned with an `active: false ` . It eventually gets deleted for real. I will reach out to the team but as for our code change here, I might try to make this specific for the organization field class.

Reply:
> I will reach out to the team 

Yeah this sounds like a bug to me, as discussed in slack.

> make this specific for the organization field class

Yeah, if we keep this change, it should be specific to the case that is affected at least.